developing/growth/README.md

```diff
@@ -0,0 +1,23 @@
+# [Tools and Practice](../README.md) / Growth
+
+## Overview
+
+When working on a project, it's important to allow time for career
+growth. Folks will have different levels of experience and will need
+to explore areas that are not directly related to completing project
+work.
+
+The experience at Truss is that this exploration winds up benefiting
+the client in the long run, both through immediate impact (e.g.
+speeding up tests) and the client getting the benefit of learning that
+happened elsewhere. Allowing time for growth ensures consistent, high
+quality work which benefits everyone.
+
+Mentoring is an important part of growth.  Truss has [shared some
+techniques for
+mentoring](https://truss.works/blog/2016/8/12/theres-a-method-to-the-mentoring-advice-from-a-teacher-turned-developer).
+
+To ensure that growth happens, the time should be built into the
+estimate for a ticket. From time to time, it will be appropriate to
```

Comment:
Does this mean that we should tend to avoid learning that isn't in the service of an existing user-driven story or ticket, or that learning tasks like that should have their own ticket? This also might have implications re: billing (or does it?)

Reply:
I added a note about learning non client related things.

Reply:
Great, thank you!